### ISSUE TITLE
PHPUnit 7 dependencies issue

### ISSUE THREAD

**Issue by richardhughes** [sebastianbergmann/phpcpd#155]
Get the following when trying to install PHPUnit 7 

```
Problem 1
    - Installation request for sebastian/phpcpd ^3.0 -> satisfiable by sebastian/phpcpd[3.0.0, 3.0.1, 3.0.x-dev].
    - Conclusion: don't install phpunit/php-timer 2.0.0
    - phpunit/phpunit 7.0.0 requires phpunit/php-timer ^2.0 -> satisfiable by phpunit/php-timer[2.0.0, 2.0.x-dev].
    - phpunit/phpunit 7.1.x-dev requires phpunit/php-timer ^2.0 -> satisfiable by phpunit/php-timer[2.0.0, 2.0.x-dev].
    - phpunit/phpunit 7.0.x-dev requires phpunit/php-timer ^2.0 -> satisfiable by phpunit/php-timer[2.0.0, 2.0.x-dev].
    - Conclusion: don't install phpunit/php-timer 2.0.x-dev
    - Installation request for phpunit/phpunit ^7 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.x-dev, 7.1.x-dev].
```

Will submit a PR later to upgrade the composer package to use php-timer 2.0

**Comment by sebastianbergmann** [comment]
This is why you should use a PHAR for tools such as PHPCPD (and not install them using Composer).

**Comment by sebastianbergmann** [comment]
Fixed by PHPCPD 4.0.0.
